20.04/Dockerfile

```diff
@@ -45,7 +45,11 @@ ENV PATH=/home/circleci/bin:/home/circleci/.local/bin:$PATH \
 	LANGUAGE=en_US:en \
 	LC_ALL=en_US.UTF-8
 
-RUN apt-get update && apt-get install -y \
+RUN noInstallRecommends="" && \
+	if [[ "20.04" == "22.04" ]]; then \
+		noInstallRecommends="--no-install-recommends"; \
+	fi && \
```

Comment:
@felicianotech @BytesGuy Why exactly is this `if` necessary?
`if [[ "20.04" == "22.04" ]];` will always be true

Reply:
@luzfcb The file you are commenting on is generated by the Dockerfile template: https://github.com/CircleCI-Public/cimg-base/blob/main/Dockerfile.template#L49

`if [[ "%%VERSION_FULL%%" == "22.04" ]];` in this case resolves to `if [[ "20.04" == "22.04" ]];`